index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3da71c9faf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style/main.css">
+    <script src="./javascript/main.js" defer></script>
+    <title>Global Drummers Workshop</title>
+</head>
+
+<body class="lato-font">
+    <header id="nav-container">
+        <a class="logo-container" href="/index.html">
+            <img class="logo" src="./assets/icons/Drum-festival-logo.svg" alt="Drum Festival Logo">
+        </a>
+        <nav class="navbar">
+            <ul class="navlinks-wrapper">
+                <li>
+                    <a href="./html/about.html">About</a>
+                </li>
+                <li>
+                    <a href="#programs">Program</a>
+                </li>
+                <li>
+                    <a href="#Sponsors">Sponsor</a>
+                </li>
+                <li>
+                    <a href="#performing-artist">News</a>
+                </li>
+            </ul>
+            <button type="button" class="nav-join-btn">
+                Join Drum Festival
+            </button>
+            <div class="hamburger-btn">
+                <img class="hamburger" src="./assets/icons/menu-icon.png" alt="hamburger icon">
+            </div>
+        </nav>
+    </header>
+    <div class="back-drop">
+        <div class="faded-backdrop"></div>
+        <div class="headline-bg-image"></div>
+        <!-- <img src="./assets/images/DSC_1494-edited-1.jpg" alt="Chris Coleman"> -->
+    </div>
+    <section id="headline">
+
+        <div class="orange-red lato-font">"Hello! Drummers!!"</div>
+        <h1 class="headline-main-text orange-red cocogoose-font">GLOBAL DRUM FESTIVAL & WORKSHOP 2022</h1>
+        <p class="black-like lato-font home-headline-body">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Quae, temporibus?
+            Aliquid reiciendis adipisci error voluptatibus illo quos unde, tempore, voluptate porro id velit. Ea
+            voluptatibus maiores nemo!</p>
+        <h3 class="black-like cocogoose-font event-date">15.09.2022(THU) ~ 17(SAT)</h3>
+        <p class="black-like lato-font venue">@ O2 Arena, London, United Kingdom</p>
+    </section>
```

Comment:
- [ ] Kindly make sure your design is consistent in all page widths, currently, it shifts depending on width, making it hard to determine your intended design. Here is a [link](https://www.loom.com/share/0814161ed120414caf484e0ed0fa962c) to the video showing the change.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: 
Also, the width is opposed to change on moving to another width :heavy_check_mark: